src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import styled from 'styled-components';
+import {MdMenu, MdSearch, MdPersonOutline} from 'react-icons/md';
+
+const ContainerHeader = styled.header`
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top:0;
+  width: 100%;
+  height: 65px;
+  background-color: #ACDEDF;
+  
+`;
+
+const MenuButton = styled.button`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 55px;
+  height: 55px;
+  padding: 10px;
+  background-color: transparent;
+  border:none;
+  font-size: 1.3rem;
+`;
+
+const InputSearch = styled.input`
+  align-items: center;
+  padding: 10px;
+  background-color: transparent;
+  box-sizing: content-box;
+  padding: 10px;
+  box-sizing: content-box;
+  border: 1px solid #bbc1bf;
+  font-size: 1.3rem;
+  background: #0000;
+  border-radius: 3px;
+  height: 1rem;
+  font-size: 1rem;
+  padding-left: 30px;
+`;
+
+const DivSearch = styled.div`
+  cursor: text;
+  display: inline-flex;
+  position: relative;
+  font-size: 1rem;
+  align-items: center;
+  height: 100%;
+  justify-content: center;
+`;
+
+const DivIcon = styled.div`
+  justify-content: center;
+  color: #fff;
+  position: relative;
+  left: 30px;
+  height: 100%;
+  align-items: center;
+  display: flex;
+  font-size: 1.3rem;
+`;
```

Comment:
Please extract all the styles into different files depending of the component. You have various components mixed on the same file. I would create i.e. a file called `SearchBar.styled.js`, inside the file I inserted all the related styles for the SearchBar, export all the styles, and in another file `SearchBar.component.jsx` use them. 

You have all the components in the Header component. React's good practice is to create small components, so you could reuse them in different places of your app, if you keep them in the same it is difficult to keep track where the component is defined and how to reuse it.